src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -534,6 +534,24 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
                 }
             },
 
+            type: {
```

Comment:
But can we instrument Promise returned by `fetch` and replace response property there?